docs/development/config-files.md

```diff
@@ -197,3 +197,18 @@ In order to get your configuration file included in this library, two things mus
 1. Update the index file using `yarn run config import -i`.
 1. Check your new or changed files for potential problems using `lerna run lint:config`. Warnings in your file may be tolerated if there is a good reason for them. Errors must be fixed.
 1. Create a PR.
+
+If your device is already present in [OZW devices DB](https://github.com/OpenZWave/open-zwave/tree/master/config), you can import it by using its device id (you can find it in zwavejs2mqtt control panel) with this command: 
```

Comment:
We should also mention that the config files contains the device ID aswell, only formatted slighly diffent:
`0x0086-0x0075-0x0004` is `0086:0075:0004` over there.